chemprop/v2/featurizers/reaction.py

```diff
@@ -82,6 +82,18 @@ def mode(self) -> RxnMode:
     def mode(self, m: str | RxnMode):
         self.__mode = RxnMode.get(m)
 
+    def __call__(
+        self,
+        rxn: tuple[Chem.Mol, Chem.Mol],
+        atom_features_extra: np.ndarray | None = None,
+        bond_features_extra: np.ndarray | None = None,
+    ) -> MolGraph:
+        return self.featurize(
+            rxn=rxn,
+            atom_features_extra=atom_features_extra,
+            bond_features_extra=bond_features_extra,
+        )
+
     def featurize(
```

Comment:
I think the `featurize` method is a vestige of a different commit. Just rename `featurize` to `__call__`